### ISSUE TITLE
Autodocumentation of class property without default discards following comment.

### ISSUE THREAD

**Issue by BuildingAtom** [sphinx-contrib/matlabdomain#152]
When using `autoclass` with a class that has properties without a default instance associated to it deletes the next comment that appears. As a short term workaround, I can add a trailing comment character.

Quick example
```matlab
classdef PropertyClass < handle
    properties
        % Document this property
        documented double

        % This line is deleted
        % This line documents another property
        another_property char
    end
end
```

and the following rst file
```rst
.. mat:autoclass:: PropertyClass
   :show-inheritance:
   :members:

```

This happens with or without any sort of property validation. I can workaround it by adding a trailing comment.
```matlab
classdef PropertyClass < handle
    properties
        % Document this property
        documented double %

        % This line is no longer deleted
        another_property char %
    end
end
```

**Comment by joeced** [comment]
Tanks a lot for the detailed report. I'll look into soon

**Comment by joeced** [comment]
Hi

I cannot reproduce this. I tried it with test "test_docs" and your example and it renders:

![billede](https://user-images.githubusercontent.com/22655178/217080330-9c4a74da-f381-43ef-9dba-adbf3c22f896.png)

Can you upload a small example project where this happens.

**Comment by BuildingAtom** [comment]
Hi joeced, sorry for taking a bit to respond.

I put together a mini example and attached it here. It includes a Dockerfile and script to build in that environment, though it's nothing more than just adding the rtd theme and matlab-sphinx on top of the base sphinx docker image. To build, just run `./build-docs.sh html` in bash or similar.

[example_magic.zip](https://github.com/sphinx-contrib/matlabdomain/files/10747780/example_magic.zip)

**Comment by joeced** [comment]
I tried again - I cannot reproduce the error you are seeing. Could you try to type `pip freeze`, and post the output. This way I can see the versions of Sphinx you are using.

**Comment by BuildingAtom** [comment]
Here's the resulting output of `pip freeze`: 
[requirements.txt](https://github.com/sphinx-contrib/matlabdomain/files/10914612/requirements.txt)

**Comment by joeced** [comment]
Same result. I cannot reproduce this. Could you try do 

```
make clean
make html
```

And verify that you are really seeing this error.

**Comment by BuildingAtom** [comment]
Sure, I took a screenshot with the project I shared earlier. Both `src.PropertyClass` and `src.PropertyClassMagic` should render the exact same documentation. `src.PropertyClassMagic` shows the expected output for both. It uses the workaround where `%` is appended as a discarded comment so that the previously discarded comment appears.
![image](https://user-images.githubusercontent.com/13129304/224744961-617aecf4-6be3-4b3c-8294-a9a14d9cb46a.png)

Some additional environment details:
I'm running Windows 11, with a WSL2 Ubuntu 20.04.5 environment. I have docker installed in this environment and am running with the build_script I provided. This build script runs `make`, so I ran `./build_script.sh clean` and `./build_script.sh html` for the same effect. The output when run is as follows:
![image](https://user-images.githubusercontent.com/13129304/224746198-1eea3f69-3f19-4bda-b44e-e0160306f4a3.png)

Running from the WSL filesystem or from mounted windows filesystem doesn't seem to have an effect, nor does the EOL type being CRLF or LF.
